build-system/server.js

```diff
@@ -646,12 +646,12 @@ app.get(['/dist/cache-sw.min.html', '/dist/cache-sw.max.html'], function(req, re
  */
 function replaceUrls(mode, file) {
   if (mode) {
-    file = file.replace('https://cdn.ampproject.org/viewer/google/v5.js', 'https://cdn1.ampproject.org/viewer/google/v5.js');
```

Comment:
I added the code below for general viewer url:
`file = file.replace(/(https:\/\/cdn.ampproject.org\/viewer\/.+).max.js/g, '$1.js');`